apps/web/src/features/cohorts/components/TeacherCohorts.tsx

```
@@ -0,0 +1,573 @@
+							cohortsWithStats.map((cohort) => {
+								const enrollmentCount = cohort.activeEnrollments || 0;
+								const maxStudents = cohort.max_students || 0;
+								const progressPercentage =
+									maxStudents > 0 ? (enrollmentCount / maxStudents) * 100 : 0;
+
+								return (
+									<TableRow
+										key={cohort.id}
+										className="cursor-pointer transition-colors duration-150 hover:bg-muted/50"
+										onClick={() => window.location.href = `/admin/cohorts/${cohort.id}`}
+									>
```

Biome (2.1.2)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)